theme/tuxlite_zf/templates/index.html

```diff
@@ -19,8 +19,8 @@
 <div style="width:45%; float: right;">
 <h2>Upcoming</h2>
 
-<h3><a href="http://conference.scipy.org/scipy2015">SciPy 2015</a></h3>
-<p>Austin, TX, July 6-11, 2015</p>
+<h3><a href="http://www.scipy2016.scipy.org">SciPy 2016</a></h3>
```

Comment:
This link seems broken, it brings me to https://www.etouches.com/event-software/ ???